scripts/eachdist.py

```diff
@@ -205,6 +205,7 @@ def setup_instparser(instparser):
 
     setup_instparser(instparser)
     instparser.add_argument("--editable", "-e", action="store_true")
+    instparser.add_argument("--testdeps", action="store_true")
```

Comment:
should this be called "--with-test-deps", to match the convention of "--with-dev-deps"?